geolocator_platform_interface/lib/src/enums/location_accuracy_status.dart

```diff
@@ -0,0 +1,8 @@
+/// Represent the current Location Accuracy Status on iOS devices
+enum LocationAccuracyStatus {
+  /// A approximate location will be returned (Approximate location)
```

Comment:
nit: missing period:

```suggestion
  /// A approximate location will be returned (Approximate location).
```